package.json

```diff
@@ -47,12 +47,11 @@
         "node": ">=10.0.0"
     },
     "dependencies": {
-        "ilib": "^14.18.0"
+        "loctool": "2.23.1",
```

Comment:
You can't move loctool to the dependencies because that will create a circular dependency loctool -> plugin -> loctool

Instead, if you want to document it properly, it should go into the peerDependencies to ensure that it has the right version of loctool available

Reply:
ah... The reason why I moved to loctool to dependencies was
packages from devDependencies  didn't install properly on webOS. it looks like only installed pkg list from dependencies.
If so I'll move the loctool to something like below, 
then will check see if webOS build. this change work for webos-appinfo, webos-javascipt, webos-json-resources to check one app first.
```
 "peerDependencies": {
        "loctool": "2.23.1"
    },
```

Reply:
I moved to loctool to `peerDepencies`
and did same work to webos-appinfo-json, webos-javascript, and webos-json-resource plugin and release for quick test. (e.g https://github.com/iLib-js/ilib-loctool-webos-javascript/pull/62)
it seems ok on running webOS. I checked one app case. so I'm going to apply same work to other webos plugin for full webos build.

By the way, i think i need to consider maintaining two branch for master and develop. cause if loctool is written ''peerDependencies` it's missing in npm install list   which cases errors running test files.

Reply:
You can have it both in peerDependencies and devDependencies

Reply:
oh! I updated it. I will apply same changes to appinfo, javascript, json-resource plugins as well. and will re-verity on webOS just in case.